Css frameworks

HGarden opened the pull request:
[Page design.pdf](https://github.com/HGarden/css-frameworks-ca/files/11127224/Page.design.pdf) The Design prototype

How do i get the footer to stay at the bottom without using position sticky, when using absolute position and bot 0 it stays at the bottom but when changing size of window it displaces. 

on the profile page how do i reduce the gap between the profile and posts.

HGarden committed:
added bootstrap scss

HGarden committed:
added code

cnnrbrn commented:
- The pull request should be made to the original repo that you cloned from. It's fine for the CA to leave it like this though.
- You can import your custom component scss after the bootstrap import. It's just the bootstrap variables that need to be imported before. Though if you saw the live video I did the same thing by mistake.
- The links can also just be to "/profile", "/feed" and "/" (for the home link)
- The live-server package is missing from your devDependencies
- You always want to style the active menu item in your nav, e.g. if you are on the feed page style that menu item as active
- With your footer set to position:fixed it overlaps some of the content on mobile

HGarden committed:
Implemented feedback

Shortened some links, added live-server as dev dependency, styled active nav link, fixed footer on auth page

HGarden committed:
Delete test.html

HGarden committed:
added a post creation form to the feed page
